packages/fxa-settings/src/components/PageChangePassword/index.tsx

```diff
@@ -92,7 +94,9 @@ export const PageChangePassword = ({}: RouteComponentProps) => {
       navigate(HomePath);
     },
     onError: (e) => {
-      if (e.errno === 103) {
+      if (e.errno === AuthUiErrorNumbers.PASSWORDS_MUST_BE_DIFFERENT) {
+        setNewPasswordErrorText(e.message);
+      } else if (e.errno === 103) {
```

Comment:
it seems like we should be able to keep it at this component level...

```diff
diff --git a/packages/fxa-settings/src/components/PageChangePassword/index.tsx b/packages/fxa-settings/src/components/PageChangePassword/index.tsx
index 42a3935d2..fa55535ec 100644
--- a/packages/fxa-settings/src/components/PageChangePassword/index.tsx
+++ b/packages/fxa-settings/src/components/PageChangePassword/index.tsx
@@ -116,6 +116,9 @@ export const PageChangePassword = ({}: RouteComponentProps) => {
       )}
       <form
         onSubmit={handleSubmit(({ oldPassword, newPassword }) => {
+          if (oldPassword === newPassword) {
+            return setNewPasswordErrorText('wtf, dude?')
+          }
           changePassword.execute(
             primaryEmail.email,
             oldPassword,
```

maybe?